jetty-ee10/jetty-ee10-webapp/src/main/java/org/eclipse/jetty/ee10/webapp/WebInfConfiguration.java

```diff
@@ -80,14 +84,6 @@ public void configure(WebAppContext context) throws Exception
     @Override
     public void deconfigure(WebAppContext context) throws Exception
     {
```

Comment:
Seems a little asymmetric that we're causing the directory to be created in `WebInfConfiguration.preconfigure`, but in `WebInfConfiguration.deconfigure` we do nothing and do it in `ContextHandler.doStop` instead. Are there any implications for the timing changes of when `WebIfConfigurion.deconfigure` is called vs when `ContextHandler.doStop` is called?

Reply:
It is not asymmetric.  The `ContextHandler` always creates and deletes any temporary directory. All `WebInfConfiguration.preconfigure` does is ensure that it is created before it unpacks the webapp into it.